alpha: 0.1
beta:  2.0
kappa: 0.0

noise_std:
  process:
    acc:
      x: 0.5
      y: 0.5
      z: 0.5
    gyro:
      x: 0.05
      y: 0.05
      z: 0.05
    acc_bias:
      x: 0.01
      y: 0.01
      z: 0.01
    gyro_bias:
      x: 0.01
      y: 0.01
      z: 0.01
  vio_yaw_drift: 0.01
